Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp

```diff
@@ -121,7 +121,7 @@ simData::ObjectId addPlatform(simData::DataStore& dataStore, double lat, double
 class FindNodeByName : public osg::NodeVisitor
 {
 public:
-  explicit FindNodeByName(const std::string& name)
+  FindNodeByName(const std::string& name)
```

Comment:
well this (and 3 others you updated) should still be explicit..

Reply:
why?

Reply:
It's our code style w.r.t. single parameter constructors, unless there's a good reason for a constructor to not be explicit. Is there a benefit to a non-explicit constructor on these four classes?

Reply:
Brevity. But I will put it back; did not realize it was your code style.